docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+
+* the `liqo-demo` namespace, and the contained resources, are offloaded only to the clusters with the `topology.liqo.io/type=destination` label.
+* the pods living in the `liqo-demo` namespace only on virtual nodes.
+
+The *NamespaceOffloading* resource created by *liqoctl* in the `liqo-demo` namespace exposes the status of the offloading process, including a global *OffloadingPhase*, which is expected to be `Ready`, and a list of *RemoteNamespaceConditions*, one for each remote cluster.
+
+In this case both the *destination1* and *destination2* clusters have been selected to offload the namespace `liqo-demo`.
+
+```bash
+kubectl get namespaceoffloadings offloading -n liqo-demo -o yaml
+```
+
+```yaml
+...
+status:
+  observedGeneration: 1
+  offloadingPhase: Ready
+  remoteNamespaceName: liqo-demo
+  remoteNamespacesConditions:
+    destination1-4591d5:
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: The remote cluster has been selected through the ClusterSelector
+        field
+      reason: ClusterSelected
+      status: "True"
+      type: OffloadingRequired
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: Namespace correctly offloaded to the remote cluster
+      reason: NamespaceCreated
+      status: "True"
+      type: Ready
+    destination2-9ea373:
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: The remote cluster has been selected through the ClusterSelector
+        field
+      reason: ClusterSelected
+      status: "True"
+      type: OffloadingRequired
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: Namespace correctly offloaded to the remote cluster
+      reason: NamespaceCreated
+      status: "True"
+      type: Ready
+```
+
+Indeed, if you query for the namespaces in the *destination1* or *destination2* cluster, you should see the following output, confirming that the remote namespace has been correctly created by Liqo:
+
+```bash
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION1"
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION2"
+```
+
+Both the commands should return the following output:
+
+```text
+NAME        STATUS   AGE
+liqo-demo   Active   70s
+```
+
+## Deploy applications
+
+Now it's time to deploy the application.
+
+In order to create a replica of the application in every *destination* cluster is necessary to enforce these conditions.
+
+* The *deployment* resource must produce at least one *pod* for each *destination* cluster.
+* Each *destination* cluster must schedule at most one *pod* on its nodes.
+
+To get this result you can exploit these *kubernetes* features:
+
+* Set a number of replicas in the *deployment* which is equal to the number of *destination* clusters
+* Set *topologySpreadConstraints* inside the *deployment*'s template, which sets the *maxSkew* equal to **1**.
+
+The file `manifests/deployment.yaml` contains an example of a *deployment* which satisfies these conditions.
```

Comment:
Not clear to me. It is `manifests/deployment.yaml` a working link?

Reply:
No, is a file contained inside the example folder, I added a `./` in front of the path to make it more clear and compliant with the rest of the documentation.